Sources/TuistKit/Commands/GraphCommand.swift

```diff
@@ -21,6 +21,12 @@ public struct GraphCommand: AsyncParsableCommand, HasTrackableParameters {
         )
     }
 
+    @Option(
+        name: [.customShort("x"), .customLong("exclude-targets-containing")],
+        help: "Exclude targets that contain the specified string from the graph rendering."
+    )
+    var excludeTargetsContaining: [String]
```

Comment:
I would call the CLI parameter just `exclude`.

Reply:
Agreed. Shorter is better. Implemented the change
0f62e6e29bf98aa7ac37ca8c4d574e14c89a5bbb